dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogCustomer/PriceTiersTest.php

```diff
@@ -0,0 +1,230 @@
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Store/_files/second_store_with_second_currency.php
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/Catalog/_files/product_simple.php
+     */
+    public function testSecondStoreViewWithCurrencyRate()
+    {
+        /** @var string $storeViewCode */
+        $storeViewCode = 'fixture_second_store';
+        /** @var StoreRepositoryInterface $storeRepository */
+        $storeRepository = $this->objectManager->get(StoreRepositoryInterface::class);
+        /** @var float $rate */
+        $rate = $storeRepository->get($storeViewCode)->getCurrentCurrencyRate();
+        /** @var string $productSku */
+        $productSku = 'simple';
+        /** @var string $query */
+        $query = $this->getProductSearchQuery($productSku);
+        /** @var array $headers */
+        $headers = array_merge(
+            $this->getHeaderAuthorization('customer@example.com', 'password'),
+            $this->getHeaderStore($storeViewCode)
+        );
+
+        $response = $this->graphQlQuery(
+            $query,
+            [],
+            '',
+            $headers
+        );
+
+        $itemTiers = $response['products']['items'][0]['price_tiers'];
+        $this->assertEquals(2, sizeof($itemTiers));
+        $this->assertEquals(round(8 * $rate, 2), $this->getValueForQuantity(2, $itemTiers));
+        $this->assertEquals(round(5 * $rate, 2), $this->getValueForQuantity(5, $itemTiers));
+    }
+
+    /**
+     * @param float $quantity
+     * @param array $tiers
+     * @return float
+     */
+    private function getValueForQuantity(float $quantity, array $tiers)
+    {
+        $filteredResult = array_values(array_filter($tiers, function ($tier) use ($quantity) {
+            if ((float)$tier['quantity'] == $quantity) {
+                return $tier;
+            }
+        }));
+
+        return (float)$filteredResult[0]['final_price']['value'];
+    }
+
+    /**
+     * @param ProductInterface $product
+     * @param array $tierPriceData
+     */
+    private function saveTierPrices(ProductInterface $product, array $tierPriceData)
+    {
+        /** @var array $tierPrices */
+        $tierPrices = [];
+        /** @var ProductTierPriceInterfaceFactory $tierPriceFactory */
+        $tierPriceFactory = $this->objectManager->get(ProductTierPriceInterfaceFactory::class);
+
+        foreach ($tierPriceData as $tierPrice) {
+            $tierPrices[] = $tierPriceFactory->create(
+                [
+                    'data' => $tierPrice
+                ]
+            );
+        }
+
+        $product->setTierPrices($tierPrices);
+        $product->save();
+    }
+
+    /**
+     * @param string $productSku
+     * @return string
+     */
+    private function getProductSearchQuery(string $productSku): string
+    {
+        return <<<QUERY
+{
+  products(search: "{$productSku}") {
```

Comment:
It might be safer to use "filter sku equals" to make sure you get an exact match